src/main/java/net/neoforged/neoforge/registries/DeferredRegister.java

```diff
@@ -648,6 +663,29 @@ public <D> DeferredHolder<DataComponentType<?>, DataComponentType<D>> registerCo
         }
     }
 
+    /**
+     * Specialized DeferredRegister for {@link EntityType EntityTypes}.
+     */
+    public static class EntityTypes extends DeferredRegister<EntityType<?>> {
```

Comment:
I wonder if it should be called `Entities` to match `DataComponents` above.

Reply:
I called it `$EntityTypes` as that is what the test framework class calls them; however I can rename if desired.